src/DssSpell.sol

```diff
@@ -15,23 +15,32 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 pragma solidity 0.6.12;
-
-// Enable ABIEncoderV2 when onboarding collateral
+// Enable ABIEncoderV2 when onboarding collateral through `DssExecLib.addNewCollateral()`
 // pragma experimental ABIEncoderV2;
+
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
+import "dss-interfaces/dapp/DSTokenAbstract.sol";
```

Comment:
Probably just want to add a lightweight interface here with just `approve()` to avoid importing a full token.

Reply:
Resolving this